cylc/flow/workflow_db_mgr.py

```diff
@@ -742,8 +742,8 @@ def upgrade_pre_810(pri_dao: CylcWorkflowDAO) -> None:
 
         # We can't upgrade if the flow_nums in task_states are not
         # distinct.
-        from cylc.flow.util import deserialise
-        flow_nums = deserialise(conn.execute(
+        from cylc.flow.util import deserialise_set
```

Comment:
Already imported
```suggestion
```

Reply:
Actually it wasn't already imported. But I'm not sure why I did it at this point in the file!